dockerfile/Dockerfile

```diff
@@ -0,0 +1,69 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2016 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM ubuntu:16.04
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+ENV DEBIAN_FRONTEND noninteractive
+
+# defined arguments and defaults
+ARG USER=wso2user
+
+ARG FILES=./files
+ARG USER_HOME=/home/wso2user
+ARG WSO2_SERVER=wso2das
+ARG WSO2_SERVER_VERSION=3.1.0
+ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}.zip
+ARG JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
+ARG JAVA_HOME=${USER_HOME}/java
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends --no-install-suggests \
+       	zip \
+        unzip \
+        telnet \
+        iproute2 \
+	sudo && rm -rf /var/lib/apt/lists/*
+
+# creates the user
+RUN useradd --system --create-home --home-dir ${USER_HOME} --no-log-init ${USER}
+
+# copy the jdk and product packs
+COPY ${FILES}/${WSO2_SERVER_PACK} ${FILES}/${JDK_ARCHIVE} ${USER_HOME}/
+
+# unzip the jdk and product packs and set the permissions
+RUN unzip -q ${USER_HOME}/${WSO2_SERVER_PACK} -d ${USER_HOME}/ \
+        && mkdir -p ${JAVA_HOME} \
+        && tar -xf ${USER_HOME}/${JDK_ARCHIVE} -C ${JAVA_HOME} --strip-components=1 \
+        && rm ${USER_HOME}/${WSO2_SERVER_PACK} \
+        && rm ${USER_HOME}/${JDK_ARCHIVE} \
+	    && chgrp -R 0 ${USER_HOME} \
+	    && chown -R ${USER} ${USER_HOME} \
+	    && chmod -R 0774 ${USER_HOME}
+
+# set the user and work directory
+USER ${USER}
+WORKDIR ${USER_HOME}
+
+# set java home and path variables
+ENV HOME=${USER_HOME} JAVA_HOME=${JAVA_HOME} PATH=$JAVA_HOME/bin:$PATH USER_HOME=${USER_HOME} \
+        WSO2_SERVER=${WSO2_SERVER} WSO2_SERVER_VERSION=${WSO2_SERVER_VERSION} USER=${USER}
+
+# expose ports
+EXPOSE 9763 9443 9160 7711 7611
```

Comment:
What would be the purpose of 9160 port?

Reply:
WSO2 Data Analytics Server [documentation](https://docs.wso2.com/display/DAS310/Default+Ports+of+WSO2+Products#DefaultPortsofWSO2Products-DataAnalyticsServer) defines this as the Cassandra port on which Thrift listens to clients.